source/GameInterface/Services/Kingdoms/Messages/UpdateKingdomRelation.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Common.Messaging;
+using TaleWorlds.CampaignSystem;
+
+namespace GameInterface.Services.Kingdoms.Messages
+{
+    public record UpdateKingdomRelation : IEvent
```

Comment:
Also please add summary comments

```suggestion
    public record UpdateKingdomRelation : ICommand
```